src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.component.js

```diff
@@ -84,6 +84,7 @@ class ExistingPaymentMethodsController {
   selectPayment(){
     this.orderService.selectPaymentMethod(this.selectedPaymentMethod)
       .subscribe(() => {
+          this.orderService.storeCardSecurityCode('existing payment method');
```

Comment:
This should probably be made an exported const and used that way instead of hard coding the string everywhere.
`export const existingPaymentMethod = 'existing_payment_method';`
`include {existingPaymentMethod} from './filename'`